crates/orchestrator/src/tests/database/mod.rs

```diff
@@ -1,7 +1,61 @@
+use crate::config::config;
+use crate::jobs::types::{ExternalId, JobItem, JobStatus, JobType};
+use crate::tests::config::TestConfigBuilder;
 use rstest::*;
+use uuid::Uuid;
 
 #[rstest]
 #[tokio::test]
-async fn test_database() {
-    // TODO: write test case
+async fn test_database_connection() -> color_eyre::Result<()> {
+    TestConfigBuilder::new().build().await;
+    Ok(())
+}
+
+/// Tests for `create_job` operation in database trait.
+/// Creates 3 jobs and asserts them.
+#[rstest]
+#[tokio::test]
+async fn test_database_create_job() -> color_eyre::Result<()> {
+    TestConfigBuilder::new().build().await;
+
+    let config = config().await;
```

Comment:
you can probably have a fixture to get the config, also add the #[awt] macro [attribute](https://docs.rs/rstest/latest/rstest/attr.rstest.html#async) 
```suggestion
   #[awt]
   async fn test_database_create_job(#[future] config ) -> color_eyre::Result<()> {
```

Reply:
Added